helpers/queries/jupiter-lend-ethena.sql

```
@@ -0,0 +1,295 @@
+, juplend_protocol_tokens as (
+    select protocol_type, mint, min(block_date) as block_date
+    from (
+        select protocol_type, supply_mint as mint, block_date from juplend_protocols
+        union
+        select protocol_type, borrow_mint as mint, block_date from juplend_protocols
+    )
+    group by 1, 2
+)
+
+-- Get vault addresses and decimals for each token
+, juplend_liquidity_tokens as (
```

SQLFluff (4.1.0)

[error] Query defines CTE "juplend_liquidity_tokens" but does not use it.

(ST03)